lua/telescope/pickers/entry_display.lua

```diff
@@ -8,11 +10,19 @@ entry_display.create = function(configuration)
   for _, v in ipairs(configuration.items) do
     if v.width then
       local justify = v.right_justify
+      local s
       table.insert(generator, function(item)
+        if s == nil then
+          local status = state.get_status(vim.api.nvim_get_current_buf())
+          s = {}
+          s[1] = vim.api.nvim_win_get_width(status.results_win) - #status.picker.selection_caret
+          s[2] = vim.api.nvim_win_get_height(status.results_win)
+        end
+        local width = resolve.resolve_width(v.width)(nil, s[1], s[2])
```

Comment:
Cant we cache `[v.width] = resolve.resolve_width`? Because this is the generator function that will be called for every entry that will be displayed